onnxruntime/python/tools/quantization/operators/softmax.py

```diff
@@ -77,15 +85,11 @@ def quantize(self):
 class QDQSoftmax(QDQOperatorBase):
     def quantize(self):
         super().quantize()
-        if self.quantizer.activation_qType == onnx.onnx_pb.TensorProto.UINT8:
-            out_scale = 1 / 256.0
-            out_zero_point = 0
-        elif self.quantizer.is_activation_symmetric:
-            # results are all greater or equal to 0, so we can only use
-            # half of the range
-            out_scale = 1 / 127.0
-            out_zero_point = 0
-        else:
-            out_scale = 1 / 256.0
-            out_zero_point = -128
```

Comment:
Softmax ranges from 0.0 to 1.0. The above scale and zero-point values can be attained by setting (rmin,rmax) to (0.0,1.0) and calling `compute_scale_zp`. Therefore, the code below does exactly that, which has the added benefit of also computing the correct values for 16bit quantization.